src/archetypes/referencebrowserwidget/browser/popup.pt

```diff
@@ -97,6 +99,14 @@
                i18n:domain="plone"
                i18n:attributes="value label_search;"
                />
+  &nbsp;<span style="font-weight: bold; color: grey; border-bottom: 1px dotted; cursor: help;"
+              title="#"
+              id="searchWildcardHelp"
+              tal:condition="python: search_index in wildcardable_indexes"
+              i18n:attributes="title"
+              data-wildcardableIndexes="#"
+              tal:attributes='title python: widget.use_wildcard_search and "wild_card_search_enabled_help" or "wild_card_search_disabled_help";
+                              data-wildcardableIndexes wildcardable_indexes_as_json'>?</span>
```

Comment:
Everybody see 'wild_card_search_enabled_help' for now as it is not yet translated...  I will propose translations in "en" and "fr" in p.a.locales when this will be ok to merge or merged.

I am ok to move this to a separated method if it helps i18ndude finds it...

I will update my PR and provide a 'default' in the translation...

Gauthier